### ISSUE TITLE
Base queries with sub queries or property chains do not return any result

### ISSUE THREAD

**Issue by bertrandgorge** [Open-CSP/WikiSearch#28]
When using property chains ([doc here](https://www.semantic-mediawiki.org/wiki/Help:Subqueries_and_property_chains)), WikiSearch returns an empty result set... Here is test page: https://wiki.tripleperformance.fr/wiki/Pays_de_la_Loire

I tried with both: 

- `|base query=[[Est dans le département::<q>[[A une région::{{PAGENAME}}]]</q>]]`

and 

- `|base query=[[Est dans le département.A une région::{{PAGENAME}}]]`

```
{{#WikiSearchConfig:
|base query=[[A un type de page::Exemple de mise en œuvre]][[Est dans le département::<q>[[A une région::{{PAGENAME}}]]</q>]] 
|aggregation size=15
|A un type de page
|A un type de production
|A un objectif
|Est dans le département
|A comme modèle ESR
|A un mot-clé
|A comme agriculteur
|Est produit dans le cadre du programme
|Remarquable
|Type de bioagresseur
|A une date de mise en œuvre
|Est produit par
}}
```

**Comment by D-Groenewegen** [comment]
I can confirm the problem is still there in 4.3.0 (June 2023).

**Comment by bertrandgorge** [comment]
Yes... I've reverted the page so that you can have a look : https://wiki.tripleperformance.fr/wiki/Pays_de_la_Loire

Actually it's worse than before as it matches now basically all the pages of the wiki. 😯 Could this be linked to this one ? https://github.com/Open-CSP/WikiSearch/issues/66 (NB : I've recently upgraded the code base from 1.39 to 1.43 and this is new...!)

**Comment by bertrandgorge** [comment]
I've had a new run at it with Claude. Apparently the problem comes from the line here: 
https://github.com/Open-CSP/WikiSearch/blob/c9a94e0bbcb737460fb7b23ec933e1ed1bcfcabf/src/SearchEngineConfig.php#L197

The fact that the PPFrame::RECOVER_ORIG flag is passed blocks the parameter from being fully expanded... The fix is simply to remove the flag. It worked for me !

See https://github.com/Open-CSP/WikiSearch/pull/88
